### PR TITLE
Update licensing-model-azure-hybrid-benefit-ahb-change.md

### PR DESCRIPTION
**Pull request by ujpat**

### CODE REVIEW

**Comment by ujpat** [comment]
Documenting based on 

PS /home/yogaprakash> New-AzSqlVM -Name $vm.Name -ResourceGroupName $vm.ResourceGroupName -Location $vm.Location
New-AzSqlVM_CreateExpanded: /usr/local/share/powershell/Modules/Az.SqlVirtualMachine/2.0.0/custom/New-AzSqlVM.ps1:626
Line |
626 |          Az.SqlVirtualMachine.internal\New-AzSqlVM @PSBoundParameters
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Please specify license type for Sql Virtual Machine. The property is 'SqlServerLicenceType' and allowed values are PAYG, AHUB and DR.
PS /home/yogaprakash> New-AzSqlVM -Name $vm.Name -ResourceGroupName $vm.ResourceGroupName -Location $vm.Location -LicenseType AHUB                                                                                 
New-AzSqlVM_CreateExpanded: /usr/local/share/powershell/Modules/Az.SqlVirtualMachine/2.0.0/custom/New-AzSqlVM.ps1:626
Line |
626 |          Az.SqlVirtualMachine.internal\New-AzSqlVM @PSBoundParameters
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | SQL Evaluation version does not support licensing changes.

**Comment by Court72** [comment]
@bluefooted

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Update-AzSqlVM -ResourceGroupName <resource_group_name> -Name <VM_name> -License
 
 Changing the license model is:
 
-- Only supported for the Standard and Enterprise editions of SQL Server. License changes for Express, Web, and Developer are not supported.
+- Only supported for the Standard and Enterprise editions of SQL Server. License changes for Express, Web, and Developer and Evaluation are not supported.
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md]:
```suggestion
- Only supported for the Standard and Enterprise editions of SQL Server. License changes for Express, Web, Developer, and Evaluation are not supported.
```

**Review by bluefooted — APPROVED** [review]

**Comment by bluefooted** [comment]
#sign-off

**Pull request merged**